diff -urN libmovtar-0.1.3.orig/configure.in libmovtar-0.1.3/configure.in
--- libmovtar-0.1.3.orig/configure.in	Sun Apr 27 10:39:06 2003
+++ libmovtar-0.1.3/configure.in	Sun Apr 27 10:55:37 2003
@@ -46,9 +46,10 @@
   mips*) 
    echo Activating IRIX changes !
    CFLAGS="$CFLAGS -DIRIX -DWORD_BIGENDIAN";;
+  powerpc*)
+   CFLAGS="$CFLAGS -DIRIX";;    
   i?86*) 
-   echo Activating Intel MMX !
-   RTJPEG_MMX="-DMMX";;
+   AC_ARG_WITH(mmx, [  --with-mmx  Use MMX in RTjpeg (no)], RTJPEG_MMX="-DMMX", RTJPEG_MMX="");;
 esac
 
 dnl madmac/gernot: 
diff -urN libmovtar-0.1.3.orig/movtar_play.c libmovtar-0.1.3/movtar_play.c
--- libmovtar-0.1.3.orig/movtar_play.c	Sun Apr 27 10:39:06 2003
+++ libmovtar-0.1.3/movtar_play.c	Sun Apr 27 10:40:43 2003
@@ -12,10 +12,6 @@
 #include <SDL/SDL.h>
 #include <SDL/SDL_timer.h>
 
-#ifndef IRIX
-#define JPEG_INTERNALS
-#include <jinclude.h>
-#endif
 #include <jpeglib.h>
 
 #include <glib.h>
@@ -579,7 +575,7 @@
     case 2:
 #ifndef IRIX
       cconvert = cinfo.cconvert;
-      cconvert->color_convert = ycc_rgb16_convert_mmx;
+      //cconvert->color_convert = ycc_rgb16_convert_mmx;  // where is CPU detection?
 #else
       fprintf(stderr, "15/16 bits per pixel can't be decoded by libjpeg on IRIX!");
 #endif
